### ISSUE TITLE
Accessibility improvements on Tab component

### ISSUE THREAD

**Issue by muratcorlu** [Trendyol/baklava#289]
- [ ] Tab component should be usable with keyboard navigation.
- [ ] Tab component should use related `aria-` attributes. [More info](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role)
